@here/harp-mapview/lib/MapView.ts

```diff
@@ -2713,30 +2713,35 @@ export class MapView extends THREE.EventDispatcher {
         if (this.m_drawing) {
             return;
         }
+
         ++this.m_frameNumber;
+        const frameStartTime = time;
```

Comment:
why do we have an alias for the time? Why not just rename the parameter ?

Reply:
good point. I just copy pasted stuff around. Will just rename the param.